### PR TITLE
Add additional stories

### PR DESCRIPTION
**Pull request by jasonLaster**
Adds more storybook stories:

(WIP) still in progress

Cleanup fixtures
+ Fixtures are renamed and moved
+ added fixtures/index.js

In the examples where fixtures have app state, i'm leaning towards
capturing the full app state as opposed to part of it. There are a
couple reasons:

1. other tests/stories end up needing some state
2. looking at a fixture it is hard to tell what part of the store it was
3. it's easy to capture app state: `copy(store.getState())`

Cleanup Stories / Add more stories
+ adds some simple json fixture queries for data

### CODE REVIEW

File: public/js/components/stories/Frames.js [hunk 1]
```diff
@@ -28,8 +34,8 @@ function renderContainer(Component, frames) {
       margin: "auto",
       paddingTop: "100px"
     }},
-    dom.div({ style: { border: "1px solid #ccc", padding: "20px" }},
-            createElement(Provider, { store },
-                          createElement(Component)))
+    createElement(Provider, { store },
+      dom.div({className: "accordion"},
+        dom.div({className: "_content"}, createElement(Component))))
```
Comment (jasonLaster) [inline public/js/components/stories/Frames.js]: this was needed for styles to apply. It *should* be a code smell that an element needs additional surrounding context to style correctly. I'll file a bug to fix this soon.

**Comment by jasonLaster** [comment]
@jlongster r?

I was initially going to do more work cleaning up the stories, but i think it's fine for now

File: public/js/components/stories/Frames.js [hunk 2]
```diff
@@ -28,8 +34,8 @@ function renderContainer(Component, frames) {
       margin: "auto",
       paddingTop: "100px"
     }},
-    dom.div({ style: { border: "1px solid #ccc", padding: "20px" }},
-            createElement(Provider, { store },
-                          createElement(Component)))
+    createElement(Provider, { store },
+      dom.div({ className: "accordion" },
```
Comment (jlongster) [inline public/js/components/stories/Frames.js]: Why are you wrapping this with an accordion class? If these are to be as docs that's sort of confusing.

**Comment by jlongster** [comment]
r+ I like consolidating the fixtures, good idea. I think you answered my one question in an outdated commit actually. Feel free to merge.

**Pull request merged**